apps/frontend/src/context/DataContext.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { DataPoint, Lap, Waypoint } from '../types';
 import { distanceToCoordinates } from '../utils/gps';
-import { getPowerToSpeedMap } from '../utils/speed';
+import { getPowerToSpeedMap } from '@dundring/utils/src/speed';
```

Comment:
Remember to add an import+export for the speed file in `utils/src/index.ts` – that makes it possible to change this to a import from `@dundring/utils`
```suggestion
import { getPowerToSpeedMap } from '@dundring/utils';
```

Reply:
the more you know 🤩